src/scenes/Contact/styles.scss

```diff
@@ -0,0 +1,88 @@
+@import 'ambience/typography.scss';
+@import 'ambience/responsive.scss';
+
+.page-contact {
+  position: relative;
+
+  width: 100vw;
+  min-height: 80vh;
+  margin-top: 150px;
+  padding-top: 200px;
+  padding-bottom: 80px;
+
+  color: #FFF;
+  background-color: #1C281E;
+
+  & > .cloud {
+    position: absolute;
+    z-index: 1;
+    top: 0;
+
+    transform: rotate(180deg);
+  }
+
+  & > .simple-title {
+    color: #FFF;
+  }
+
+  & > .simple-title > .divisor {
+    margin-bottom: 24px;
+  }
+
+  & > .subtitle {
+    font-size: 26px;
+
+    text-align: center;
+    letter-spacing: .41px;
+
+    opacity: .8;
+  }
+
+  & > .form {
+    display: flex;
+    flex-direction: column;
+
+    width: 50vw;
+    margin: 0 auto;
+    margin-top: 43px;
+
+    @include small-only {
+      width: 82vw;
+    }
+  }
+
+  & > .form > label {
+    margin-top: 18px;
+  }
+
+  & > .form > label > input,
+  & > .form > label > textarea {
+    font-size: 16px;
+
+    box-sizing: border-box;
+    width: 100%;
+    margin-top: 8px;
+    padding: 20px;
```

Comment:
Have you tried to write something inside the form fields? This padding is making it impossible.